docs/making-your-own.md

```diff
@@ -1,39 +1,26 @@
 # Making your Own
 
-This project generates [operating system images](images.md) that you can use on your own PC.
-However, one of the main goals is to provide a custom toolkit for you to make your own custom image. 
+[Universal Blue](https://universal-blue.org/) is both a collection of native container images based on Fedora and a toolkit for you to make your own.
 
 !!! Note "This is moving fast"
 
     It's still early in the project's life, we're in need of people to help with these docs, so if you find a problem please consider [submitting a pull request](https://github.com/ublue-os/website) to help us improve.
 
 ## Why make your own image?
```

Comment:
This section seems like it's split up in an odd way.

Reply:
How about:

The concept around image based operating systems balances on the idea that the core image is ready to go, and ideally users don't need to touch it, but we understand that people like to tinker. Instead of [layering packages](https://docs.fedoraproject.org/en-US/iot/adding-layered/) that compromise the integrity of upgrades, we provide the tools for you to make your own image allowing you to have a pristine setup for the life of your hardware with minimal maintenance.